setup.cfg

```diff
@@ -18,8 +18,8 @@ platforms = OSX,Linux
 # https://pypi.python.org/pypi?%3Aaction=list_classifiers
 classifiers =
     Development Status :: 5 - Production/Stable
-    Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
Why not also add 3.11 and 3.12?

Reply:
Because I was not sure how useful it is to list versions here? but yes, now the tests passed with 3.11 and 3.12 you're right, I'll add them